src/components/Text.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react-native';
+import { Text } from './Text';
+import { BrandConfig, BrandConfigProvider } from 'src/hooks/useBrandConfig';
+import { colors } from '../theme';
+
+describe('Without configuration', () => {
+  test('Renders text with default style (baseTheme.colors.text)', () => {
+    render(
+      <BrandConfigProvider>
```

Comment:
Shouldn't need to add the provider here. It might be a good idea to remove it so we can ensure that themed components can be used with the default context value.

Reply:
Oh, good call!